translate/src/core/utils/fluent/extractAccessKeyCandidates.test.js

```diff
@@ -1,69 +1,108 @@
+import ftl from '@fluent/dedent';
 import { extractAccessKeyCandidates } from './extractAccessKeyCandidates';
 import { parseEntry } from './parser';
 
 describe('extractAccessKeyCandidates', () => {
-  it('returns null if the message has no attributes', () => {
-    const message = parseEntry('title = Title');
-    const res = extractAccessKeyCandidates(message);
+  it('returns an empty list if the message has no label attribute or value', () => {
+    const input = ftl`
+      title =
+        .foo = Bar
+        .accesskey = B
+      `;
+    const message = parseEntry(input);
+    const res = extractAccessKeyCandidates(message, 'accesskey');
```

Comment:
I would actually expect B, a, r to be the candidates here. What if we just take the attribute preceding the "accesskey" attribute in cases when none of the attribute names matches the allowed list?

Reply:
Then we would at least currently end up misleading the localiser to presume that the preceding attribute is a good candidate for access keys. I looked through all uses of `.accesskey` in m-c, and the list that's proposed here is exhaustive for our current usage.

So while using an effectively random preceding attribute would technically work and be a bit more future-proof, I'm not so sure that it would be currently useful.

Reply:
OK. You got me at "I looked through all uses of `.accesskey` in m-c".